src/sentry/static/sentry/app/constants/index.jsx

```diff
@@ -63,6 +63,8 @@ export const MENU_CLOSE_DELAY = 200;
 
 export const DEFAULT_STATS_PERIOD = '14d';
 
+export const DEFAULT_USE_UTC = true;
```

Comment:
I feel like a default prop on the time range component would be more straightforward than exporting this as a constant and then importing it everywhere we use the time range selector and passing it into that

Reply:
I moved it to be a default prop of `TimeRange` but keeping the constant since we use it elsewhere (and will be good to keep track of since we may need to move this into a user preference in the future).